packages/shared/args.js

```diff
@@ -0,0 +1,27 @@
+export function args(args) {
+    const parsedArgs = {
+        named: {},
+        positional: []
+    };
+
+    const argsLength = args.length;
+    for (let i = 0; i < argsLength; i++) {
+        const arg = args[i];
+        if (arg.indexOf("-") != -1) {
+            const name = getArgName(arg);
+            if (args[i + 1] === undefined)
+                throw Error('named argument should have a value');
```

Comment:
It would be nice to add the name of the argument with missing value to the error message.

Reply:
I've updated the message.